src/sentry/scim/endpoints/members.py

```diff
@@ -375,6 +385,9 @@ def put(self, request: Request, organization, member):
         member.flags["idp:role-restricted"] = idp_role_restricted
         member.save()
 
+        metrics.incr(
```

Comment:
We may also want to know when a role is explicitly set when creating a member.

Reply:
I added the `update_role` metric when provisioning a member w/ an explicit role